src/App.tsx

```diff
@@ -19,11 +20,10 @@ export default function App() {
             </ProtectedRoute>
           }
         />
-        {/* <ProtectedRoute component={<HomeScreen />} /> */}
```

Comment:
This line should not be deleted. Revert this line as HomeScreen is always in a protected route.